README

```diff
@@ -1578,6 +1578,11 @@ Typically these are:
     - Hardware flow control (CTS/RTS) enabled
     - Hardware flow control (DSR/DTR) disabled
 
+Furthermore, additional parameters can be specified to control the behavior
+of the softmodem connection. These are all parameters:
+
+ * connectspeed  - The speed to report upon connectionn.  Default: 57600
+
```

Comment:
The other `modem <setting:value>`'s aren't described here, so this feels a bit bolted on.

I would suggest removing this, and just mentioning it further up, something like:


![2022-07-09_17-50](https://user-images.githubusercontent.com/1557255/178127206-5e6a799d-d15e-4a07-bca0-513c2835cd93.png)